From 14dc516c7cd1306a2f3b21df50ceab5f8e6dce2f Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Tue, 6 Apr 2021 20:10:40 -0300
Subject: [PATCH] Patch sysinfo for static LTO

---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 83341b3..f35ae7d 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -24,7 +24,7 @@ repository = "https://github.com/GuillaumeGomez/sysinfo"
 
 [lib]
 name = "sysinfo"
-crate_type = ["rlib", "cdylib"]
+crate_type = ["rlib"]
 [dependencies.cfg-if]
 version = "1.0"
 
-- 
2.30.2

